packages/plugin-encryption/test/integration/spec/kms.js

```diff
@@ -15,8 +15,9 @@ describe(`Encryption`, function() {
   this.timeout(30000);
   describe(`KMS`, () => {
     let spark, user;
```

Comment:
can you name your users spock and mccoy to keep with the rest of the test suite? :)